forwardproxy.go

```diff
@@ -240,6 +249,19 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 		}
 	}
 
+	// init masque proxy
+	h.udpProxyServer = &udpProxyServer{}
+	// parse http2/http3 uri template
+	if h.URITemplate == "" {
+		h.uriTemplate = uritemplate.MustNew("https://{host}/.well-known/masque/udp/{target_host}/{target_port}/")
```

Comment:
Is it necessary to introduce a library for this? Can you use simpler functions?

Reply:
Removed